plan_runner/manipulation_station_simulator.py

```diff
@@ -59,12 +54,15 @@ def RunSimulation(self, plans_list, gripper_setpoint_list,
                       extra_time=0, real_time_rate=1.0, q0_kuka=np.zeros(7), is_visualizing=True):
         '''
```

Comment:
Use double quotes for docstrings
```suggestion
        """
```